core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.spec.ts

```diff
@@ -301,14 +302,72 @@ describe('Contributor stats component', () => {
         'lastContributedInDays',
         'role',
       ]);
+      expect(component.getOverallItemCount()).toEqual('1+');
+    }));
+
+    it('should check for upperlimit for pagination from total number of items, as its lower than items per page combined with page number.', fakeAsync(() => {
```

Comment:
Rename the test to `it('indexes last item on page as overall item count when current page's item count is less than max item count per page', ...);`

Reply:
Done. I renamed it to :-
should index last item on page as "max allowed per page item count plus previous pages item count" when "overall item count" exceeds "max allowed per page item count plus previous pages item count". 

Which is a bit of a mouthful, hopefully the quote marks help explain what is happening here. I renamed the test for the opposite case similarly.

Reply:
@stevenjgodfrey You're right. The test names are getting long. The first thing that stands out is the phrase `"max allowed per page item count plus previous pages item count"` used in these test names. Maybe our test names shouldn't include this calculation itself. Instead, the test names should include what the result of this calculation represents.

After some thinking, I've realized that the result of this calculation represents the overall item count between the table's first page and the table's current page. Maybe we can use the phrase `"current page\'s cumulative item count"` to shorten the test name a bit?

Reply:
Done